README.rst

```diff
@@ -5,9 +5,17 @@
 About
 -----
 
-This is a meta-package which can be used in ``pyproject.toml`` files to
-automatically provide the oldest supported version of Numpy without having to
-list them all. In other words::
+This is a meta-package which can be used in ``pyproject.toml`` files
+to automatically provide the oldest version of Numpy that supports the
+given Python version and platform, without having to list them all.
```

Comment:
would be good to add the bit about that oldest version having wheels for Windows/Linux/macOS available, as discussed in gh-2.

LGTM other than that

Reply:
Added mention of wheels.